pep-513.rst

```diff
@@ -394,6 +414,16 @@ and build multiple wheels in order to cover all the common Linux distributions.
 Therefore we consider such proposals to be out-of-scope for this PEP.
 
 
+Future updates
+==============
+
+We anticipate that at some point in the future there will be a
+``manylinux2`` specifying a more modern baseline environment (perhaps
+based on CentOS 6), and someday a ``manylinux3`` and so forth, but we
+defer specifying these until we have more experience with the initial
+``manylinux1`` proposal.
```

Comment:
How about mentioning a possibility for a "manylinux1.1" that would entail a possible expansion of the library list but no change in `pip`'s platform detection code or the platform tag?